README.md

```diff
@@ -24,3 +24,7 @@ MIT
 ## Author
 
 Yasuhiro Matsumoto (a.k.a. mattn)
+
+## Sound
+
+aonrjp
```

Comment:
Please add the link to download original file.

Reply:
OK.

Reply:
Prease wait...

I will add URL soon.

Reply:
Thanks for the link to wav file. What the license of the file? Whose is this?

Reply:
Thanks to reply.

"aonr.jp" create this wav file. (github url : https://github.com/greenlaver)

This sound's license is public domain.

Reply:
Thanks. Then, please update README.md like below.
```
## Thanks

sl.wav : https://aonr.jp/
```

Reply:
I fixed. Please check!